modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -2455,6 +2455,19 @@ private ObjectDefinition _publishObjectDefinition(
 			objectDefinitionPersistence, _objectDefinitionSettingLocalService,
 			_objectRelationshipLocalService, _objectRelationshipPersistence);
 
+		for (ObjectField objectField : objectFields) {
+			if (Objects.equals(
+					objectField.getDBTableName(),
+					objectDefinition.getDBTableName())) {
+
+				continue;
+			}
```

Comment:
@Aquiiiles please increment the condition to avoid updating system object fields.